client-generator/openapitools.json

```diff
@@ -8,7 +8,7 @@
       "ts-rxjs": {
         "generatorName": "typescript-rxjs",
         "output": "../client-generated/",
-        "inputSpec": "http://localhost:3000/api-json",
+        "inputSpec": "http://localhost:3001/api-json",
```

Comment:
Why this change? I believe the server is listening on port 3000 (at least in the local dev setup). The dev server for the frontend is on port 3001.

Reply:
Good catch, adapted to some local changes. Reverted in [0a8c889](https://github.com/osrgroup/msadoc/pull/83/commits/0a8c8899ae7e8d41190b6e5eac5c7b9495fd2dcf)